repo2docker/buildpacks/base.py

```diff
@@ -481,13 +482,38 @@ def render(self):
             labels=self.get_labels(),
             build_script_directives=build_script_directives,
             assemble_script_directives=assemble_script_directives,
-            build_script_files=self.get_build_script_files(),
+            build_script_files={
```

Comment:
Should probably move this to a temporary variable for clarity, and add a comment about what it's doing.

Reply:
Updated